Dockerfile

```diff
@@ -7,8 +7,8 @@ FROM ${DOCKER_IMAGE} AS builder
 
 # We don't want to run our app as root, so we define a worker user.
 ARG USERNAME=worker
-ARG USER_UID=1000
-ARG USER_GID=$USER_UID
+ARG USER_UID=1010
```

Comment:
Is this necessary because the base image already has a `USERID` of 1000?

Reply:
Yes